endpoints/cookie_sync.go

```diff
@@ -427,6 +427,21 @@ func (c *cookieSyncEndpoint) handleResponse(w http.ResponseWriter, tf usersync.S
 		})
 	}
 
+	// Debug Flag Response Handling
+	if debug {
+		biddersSeen := make(map[string]bool)
```

Comment:
Please use `map[string]struct{}` as that doesn't use memory for the value (and the value doesn't matter here, just the presence of the key).